tests/integration/utils.py

```diff
@@ -275,3 +276,47 @@ def assert_content_manifest(client, request_id, image_contents):
     response_data = content_manifest_response.data
     assert_content_manifest_schema(response_data)
     assert image_contents == content_manifest_response.data["image_contents"]
+
+
+def package_manager_common_checks(test_env, env_data, tmpdir):
+    client = Client(test_env["api_url"], test_env["api_auth_type"], test_env.get("timeout"))
+    initial_response = client.create_new_request(
+        payload={"repo": env_data["repo"], "ref": env_data["ref"], "pkg_managers": env_data["pkg_manager"]}
+    )
+    completed_response = client.wait_for_complete_request(initial_response)
+    assert completed_response.status == 200
+    assert completed_response.data["state"] == "complete"
+    assert completed_response.data["state_reason"] == "Completed successfully"
```

Comment:
For instance, this could be a single function, to check if the response is complete. Ithough @ejegrova had introduced a function for this before.

Reply:
I have it in https://github.com/release-engineering/cachito/pull/292